row_generation/plain_hunt_generator.py

```diff
@@ -1,12 +1,17 @@
+""" A module to hold a row generator that produces plain hunt on any (even) stage. """
+
 from typing import List
 
-from .row_generator import RowGenerator
 from bell import Bell
 
+from .row_generator import RowGenerator
+
 
 class PlainHuntGenerator(RowGenerator):
+    """ A row generator to create plain hunt on any (even) stage. """
```

Comment:
Magically this should also works on odd stages (there's a hidden implied lieing behind on cross ...)

`PlainHuntGeneratorTests.test_singles` demonstrates this

Reply:
Possibly it's a bug that being passed `13` results in 2 and 3 crossing, though I feel that is as accurate as turning it into `123`

Either way I guess the backstroke case here should just be `[1]`

Reply:
Hmmm I'm not sure how I feel about the odd-stage cross.  I think the behaviour is so badly defined (should someone implicitly lead or lie?) that in all of my ringing code, odd-stage cross causes an exception.

Reply:
I would also say that `13` should turn into `123`, since it's the only valid way that places can be made in 1st's and 3rd's places.

Reply:
But what should `15` be?
It's a cheap and cheerful imp at the moment

Reply:
I think BellMetal will turn `15` into `145` when using it to permute changes, but this was just the easiest way to implement it.

I think the cheap and cheerful impl is fine for now - it's only getting its input from other parts of our code not the user.

Reply:
Thinking more, I have a feeling that the `PlaceNotation` part of my code will never generate `15`, and so the only way to feed the permute function with something like that is to use an `unsafe` block which is exactly as it sounds - very unsafe and used as a last resort.

But these strong compile time guarantees are a feature of rust that Python lacks, so aren't really an option.